takwimu/views.py

```diff
@@ -233,125 +233,15 @@ def get(self, request, *args, **kwargs):
                 else:
                     results.append(hit)
 
+            #limit search results to 30 items
+            if len(results) > 30:
+                results = results[:30]
```

Comment:
1. Are we using `views.py` in any way for the search? I'm assuming not so why are we adding the `limit` here?
 2. Better to `limit` the result in the search query itself and not _after_ the `ElasticSearch` has returned them. Otherwise we're making `ElasticSearch` work for nothing.